R/mia.R

```diff
@@ -322,3 +322,54 @@ NULL
 #' \url{https://static-content.springer.com/esm/art%3A10.1186%2Fs40168-020-00816-x/MediaObjects/40168_2020_816_MOESM3_ESM.docx}
 #' 
 NULL
+
+#' CRC_QMP
+#'
+#' The study combined Quantitative Microbiome Profiling (QMP) with 
+#' extensive patient phenotyping from a group of 589 colorectal cancer (CRC) 
+#' patients, advanced adenoma (AA) patients, and healthy controls. 
+#' The researchers also reanalyzed data from 15 published studies encompassing 
+#' a total of 4,439 patients and controls. By implementing confounder control and 
+#' quantitative profiling methods, the study was able to reveal potential misleading 
+#' associations between microbial markers and colorectal cancer development that 
+#' were driven by other factors like intestinal inflammation, rather than the cancer 
+#' diagnosis itself.
+#'
+#' @format A Large Tree Summarized Experiment dataset with 676 features and 589 samples. 
+#' The rowData contains Species name. The colData includes:
+#' 
+#' \describe{
+#'   \item{sample_ID}{Sample ID from the corresponding study}
+#'   \item{Diagnosis}{Diagnosis type, with possible values: ADE (advanced adenoma), 
+#'   CRC (colorectal cancer), CTL (control)}
+#'   \item{Colonoscopy}{Colonoscopy result, with possible values: FIT_Positive, 
+#'   familial_risk_familial_CRC_FCC, familial_risk_no, abdomil_complaints}
+#' }
+#'
+#' @name CRC_QMP
+#' @docType data
+#' @keywords datasets
+#' @usage data(CRC_QMP)
+#' @seealso \code{\link{mia-datasets}}
+#' @author 
+#' Tito, R. Y., et al.
+#' Raúl Y. Tito, Sara Verbandt, Marta Aguirre Vazquez, Leo Lahti, Chloe Verspecht, Verónica Lloréns-Rico, Sara Vieira-Silva,
+#' Janine Arts, Gwen Falony, Evelien Dekker, Joke Reumers, Sabine Tejpar & Jeroen Raes
+#' @references
+#' Raúl Y. Tito, Sara Verbandt, Marta Aguirre Vazquez, Leo Lahti, Chloe Verspecht, Verónica Lloréns-Rico, Sara Vieira-Silva,
+#' Janine Arts, Gwen Falony, Evelien Dekker, Joke Reumers, Sabine Tejpar & Jeroen Raes (2024). 
+#' Microbiome confounders and quantitative profiling challenge predicted microbial targets in colorectal cancer development. 
+#' Nature Medicine,30, 1339-1348. 
+#' \url{https://www.nature.com/articles/s41591-024-02963-2#Sec8}
+#' 
+#' The supplementary tables contain colonoscopy referral reasons for the LCPM cohort,
+#' LCMP cohort variables and enterotypes, continuous variables' associations with cancer 
+#' progression, categorical variables' associations with cancer progression, microbiome 
+#' variation analysis in the LCMP cohort, included and excluded species list, species 
+#' abundance differences across diagnostic groups, species abundance associations, 
+#' species abundance differences in normal calprotectin levels subset, categorical 
+#' variables' associations with enterotype distribution, species detected in negative 
+#' controls, species abundance differences between CRC and controls in published cohorts, 
+#' and absolute taxonomic abundances at species level in the LCMP cohort.
```

Comment:
Are these included in the R data set?

Reply:
No, this is the whole supplementary data information from the CRC paper, there are 14 supplementary tables. I needed to use only two from the above

Reply:
Shall I remove the table information which are unused?